### ISSUE TITLE
No autocompletion for input bar

### ISSUE THREAD

**Issue by CirrusClouds** [stumpwm/stumpwm#886]
I've been using stumpwm for a couple years now and I recently got a new computer. I loaded up my old config and noticed that I'm not getting autocompletion for the input bar anymore. I have not been able to find any information on this or found anyone who's had the same problem. I used the AUR repos to build stump instead of source code (which I did beforehand because of not using arch on my previous computer, I have no idea if this is relevant).

Also, tab completion still works. It's simply that the view for the autocompletion options doesn't show up.

**Comment by landakram** [comment]
Do autocompletions show up once you start typing? I wonder if you're running into a relatively new default behavior of stumpwm, where the input bar does not show the autocompletions view while the input string is empty.

I ran into this as well when I upgraded and introduced an option to make this behavior configurable: https://github.com/stumpwm/stumpwm/pull/865

**Comment by CirrusClouds** [comment]
Nope! It doesn't show up even while there's characters in there. I wouldn't mind so much if it was just that

**Comment by JamieWaymark** [comment]
Hi I'm the author of this issue on a different github. I can't replicate this issue anymore since I did a recompilation. I'm now getting the drop down like normal. Everything happens so much I guess!

**Comment by MIvanchev** [comment]
@dmb2 this can be closed now.
